ext/search_kit/Civi/Search/Admin.php

```diff
@@ -205,7 +205,10 @@ private static function getDefaultColumns(array $entity, iterable $getFields): a
       $possibleColumns[$column] = "$column:label";
     }
     // Other possible relevant columns... now we're just guessing
-    $possibleColumns['financial_type_id'] = 'financial_type_id:label';
+    //
+    // TODO: these can be specified using the @searchColumns annotation on
+    // the Api4 entity class so would probably be better to specify sensible
+    // options for core entities explicitly - which allows you to order logically too
```

Comment:
Agree we should get rid of this whole function in favor of explicit `@searchFields` annotations on every entity.